Week1/solution7.py

```diff
@@ -0,0 +1,10 @@
+#Write a python script to enter any string, replace vowel with its position number.
+str1=input("Write a string: ")
+str0=str1
+res=str1
+for i in str1:
+    if i=='A' or i=='E' or i=='I' or i=='O' or i=='U' or i=='a' or i=='e' or i=='i' or i=='o' or i=='u':

```

Comment:
write this statement other better way without using "or" operator.

Reply:
instead of 'or', membership operator IN can be used.
```
ch = "Hello World"
vowels = ['a', 'e', 'i', 'o', 'u', 'A', 'E', 'I', 'O', 'U']  
for i in ch:
    if i in vowels:
        pass
```